app/views/shared/vacancy/_jobseeker_view.html.slim

```diff
@@ -58,12 +70,12 @@ h2.govuk-caption-l.job-caption class="govuk-!-margin-bottom-5 govuk-!-margin-top
     = render "vacancies/job_details", vacancy: @vacancy
     = render "vacancies/overview_#{@vacancy.job_location}", vacancy: @vacancy
 
-    - if @similar_jobs.present?
+    - if current_variant?(:"2022_02_similar_jobs_position_test", :default) && @similar_jobs.present?
       section#similar-jobs.similar-jobs class="govuk-!-margin-bottom-5"
         h3.govuk-heading-m = t("jobs.similar_jobs.heading")
-        - @similar_jobs.in_groups_of(2, false).each do |row|
-          .govuk-grid-row
-            = render Jobseekers::SimilarJobComponent.with_collection(row)
+        .govuk-grid-row
+          - @similar_jobs.in_groups_of(2, false).each do |row|
+            = render Jobseekers::SimilarJobComponent.with_collection(row, classes: ["govuk-grid-column-one-half"])
```

Comment:
this new component will be used going forward regardless - could we add it to the aliases in the component helper?

Reply:
sorry its not a new component, but its been updated, i think should have an alias all the same